tools/exchange/src/main/resources/server_application.conf

```diff
@@ -18,6 +18,8 @@
   # Nebula Graph relation config
   nebula: {
     addresses: ["127.0.0.1:3699"]
+    # If the data source is Hbase, metaAddress must be configured
+    metaAddress:["127.0.0.1:3699"]
```

Comment:
`127.0.0.1:54500` is default address of the meta service?

Reply:
I forget the change the meta address's default port, it's 45500.